website/en/docs/linting/rule-reference.md

```diff
@@ -212,9 +212,10 @@ Triggers when you use `invariant` to check a condition which we know must be tru
 Note that this lint does not trigger when we know a condition is always `false`. It is a common idiom to use `invariant()` or `invariant(false, ...)` to throw in code that should be unreachable.
 
 #### `deprecated-utility` <a class="toc" id="toc-deprecated-utility" href="#toc-deprecated-utility"></a>
-Triggers when you use the `$Supertype` or `$Subtype` utility types, as these types are
-unsafe and usually just equivalent to `any`. If the utilities were being used in a sound manner, the
+Triggers when you use the `$Supertype`, `$Subtype`, or `*` (existential) utility types, as these types are
```

Comment:
I think `deprecated-type` is the lint that does this, but it isn't documented at all. From looking at the code, it looks like it only fires when it encounters `*`

Reply:
I added documentation for `deprecated-type` and cross-linked from the deprecation notice.